docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -21,16 +21,17 @@ The default Helm chart deploys the following components:
 - **QueryFrontend component** (2 replicas, maxUnavailable: 1): Manages frontend queries. Up to 1 replica can be unavailable during updates.
 - **QueryScheduler component** (2 replicas): Schedules queries.
 
+{{< admonition type="note" >}}
 It is not recommended to run scalable mode with `filesystem` storage. For the purpose of this guide, we will use MinIO as the object storage to provide a complete example. 
```

Comment:
should say microservices mode instead of scalable here, yeah?

Reply:
Great catch! the dizzyness of jumping between both deployments :D